sycl/test-e2e/ESIMD/thread_id_test.cpp

```diff
@@ -2,6 +2,7 @@
 // UNSUPPORTED: cuda || hip
 // RUN: %clangxx -fsycl %s -o %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
+// XFAIL: gpu && !esimd_emulator

```

Comment:
```suggestion
// XFAIL: linux && gpu && !esimd_emulator
```
This should fix Windows check in CI.

Reply:
Thank you, Alexey. I applied your suggestion.